### PR TITLE
A Tilt Tiltfile

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (jazzdan) [inline .vscode/launch.json]: whoops will delete this

File: Tiltfile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  local_resource("go_lint", "make lint", deps=all_go_files)
+
+def get_all_ts_files(path):
+  return str(local('cd %s && find . -type f -name "*.ts*" | grep -v node_modules | grep -v __snapshots__' % path)).split("\n")
+
+def yarn_install():
+  local_resource("yarn_install", "cd web && yarn", deps=['web/package.json', 'web/yarn.lock'])
+
+def jest(path):
+  local_resource("web_jest", "", serve_cmd="cd %s && yarn run test --bail" % path, resource_deps=["yarn_install"])
```
Comment (landism) [inline Tiltfile]: nit: `cmd` is optional and defaults to empty

```suggestion
  local_resource("web_jest", serve_cmd="cd %s && yarn run test --bail" % path, resource_deps=["yarn_install"])
```

Comment (landism) [inline Tiltfile]: I don't have strong feelings either way, but alternatively to `--bail`, could consider `--notify`, which wouldn't result in the tilt status being updated, but would at least communicate to you when your tests are failing

File: web/package.json [hunk 3]
```diff
@@ -37,6 +37,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "ci": "react-scripts test --runInBand --reporters=default --reporters=jest-junit",
+    "test-bail": "react-scripts test --bail",
```
Comment (landism) [inline web/package.json]: is this used, or left over from an earlier iteration?

**Pull request merged**